### PR TITLE
Feature pausing

### PR DESCRIPTION
**Pull request by marcelkuipers**
You can now pause by pressing.
This is very small PR but I was wondering whether you guys agreed with how I implemented it.

### CODE REVIEW

**Comment by marcelkuipers** [comment]
It seems the build is failing. I will close the PR for now

**Comment by marcelkuipers** [comment]
I updated a TC and now it's working again.

File: ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java [hunk 1]
```diff
@@ -44,8 +45,11 @@ public void initialize(AppStateManager stateManager, Application app) {
 	@Override
 	public void update(float tpf) {
 		super.update(tpf);
-		hudController.updateHUD();
-		moveMovables();
+		if (!paused) {
+	        hudController.updateHUD();
+	        moveMovables();
+
+		}
```
Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java:51]: The indentation seems off here

Comment (marcelkuipers) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java:51]: It looks fine in Eclipse though :/
I will check future PR's for this.

**Comment by NilsHullegien** [comment]
Looks good now!

**Comment by dVoorhout** [comment]
I first imagined we would be doing a pause screen with appstates, but this seems real easy and relatively clean. If it we want in appstates later on anyway I guess we'll change it then. It looks good for now.

**Pull request merged**